frontend/src/views/Collection/components/ActionButtons/components/AddButton/components/Content/index.tsx

```diff
@@ -0,0 +1,21 @@
+import { Menu, MenuItem } from "@blueprintjs/core";
+import React from "react";
+import DropboxChooser, {
+  Props as DropboxChooserProps,
+} from "src/components/DropboxChooser";
+
+interface Props {
+  addNewFile: DropboxChooserProps["onUploadFile"];
+}
+
+const Content = ({ addNewFile }: Props) => {
+  return (
+    <Menu>
+      <DropboxChooser onUploadFile={addNewFile}>
+        <MenuItem text="Add Dataset" />
```

Comment:
Currently the collection level's AddButton dropdown only has `Add Dataset` action, but we will add "Add Gene Set" action later too